scripts/gitlab-build.sh

```diff
@@ -62,13 +62,18 @@ build () {
   cargo build --target $PLATFORM --release -p ethstore-cli
   echo "Build ethkey-cli:"
   cargo build --target $PLATFORM --release -p ethkey-cli
+  echo "Build whisper-cli:"
+  cargo build --target $PLATFORM --release -p whisper-cli
+  # @TODO rename binary properly
+  mv target/$PLATFORM/release/whisper-cli target/$PLATFORM/release/whisper;
```

Comment:
If you could take this TODO @niklasad1 ?

Reply:
Fixed once #8579 is merged!